roots/quran-simple.sql

```diff
@@ -56,6 +56,16 @@ CREATE TABLE `quran_text` (
   PRIMARY KEY  (`index`)
 ) ENGINE=MyISAM DEFAULT CHARACTER SET utf8 COLLATE utf8_general_ci;
 
+DROP TABLE IF EXISTS `root_letters`;
+CREATE TABLE `root_letters` (
+  `unique_id` int(10) NOT NULL auto_increment,
+  `surah_aya_id` text NOT NULL, 
```

Comment:
We'll need to change this structure up a little bit. Remember, how are Quran text and root_letters connected? I recommend having another table connecting the two (a many-to-many relationship). Also, don't forget the foreign keys.